doc/transform_methods/hansenlaw.rst

```diff
@@ -8,20 +8,21 @@ Hansen-Law
 Introduction
 ------------
 
-The Hansen and Law transform is the work of E. W. Hansen and P.-L. Law [1].
+The Hansen and Law transform [1, 2] is a fast (linear time) Abel transform.
+ 
+From the abstract [1]:
```

Comment:
This doesn't read too well, since the "...new family of algorithms" isn't the start to a new sentence. Maybe we can make the transition smoother by writing something like: 
"In their words, Hansen and Law [1] present"

Reply:
A very good point. I like your wording. Thanks.